[TT-6486/TT-9036] update error response codes to have certificate has expired error (#2599)
TykTechnologies/tyk-docs: tyk-docs/content/error-response-codes.md

| 403  | Certificate with SHA256 $certID not allowed                                             | Certificate ID is nil or invalid. Please have a valid certificate.                                                                                                      |
| 403  | Client authorize request in with invalid redirect URI                                   | Check if Auth Redirect URI is malformed or use a valid redirect URI.                                                                                                    |
| 403  | Client TLS certificate is required                                                      | Check if theres multiple APIs on the same domain with no certificates.                                                                                                  |

> | 403  | Certificate has expired                                                                 | Please update the attached certificates that are currently valid and have not expired.                                                                                                  |

Reviewer: ```suggestion
| 403  | Certificate has expired                                                                 | Please update the certificate with one that is currently valid and has not expired.                                                                                                  |
```
